src/citrine/resources/gemtables.py

```diff
@@ -122,7 +122,9 @@ def build_versions(collection: Iterable[dict]) -> Iterable[GemTable]:
             for item in collection:
                 yield self.build(item)
 
-        return self._paginator.paginate(fetch_versions, build_versions, page, per_page)
+        return self._paginator.paginate(
+            # Don't deduplicate on uid since uids are shared between versions
```

Comment:
I'm guessing in the future we'll want to deduplicate on uid + version?

Reply:
Or improve pagination more comprehensively.